modules/ossm-installing-external-control-plane.adoc

```diff
@@ -0,0 +1,457 @@
+// This procedure is used in the following assembly:
+// * service-mesh-docs-main/install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-external-control-plane_{context}"]
+= Installing the control plane and data plane on separate clusters
+
+Install {istio} on a control plane cluster and a separate data plane cluster. This installation approach provides increased security.
+
+[NOTE]
+====
+You can adapt these instructions for a mesh spanning more than one data plane cluster. You can also adapt these instructions for multiple meshes with multiple control planes on the same control plane cluster.
+====
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator on the control plane cluster and the data plane cluster.
+
+* You have `istioctl` installed on the laptop you will use to run these instructions.
+
+.Procedure
+
+. Create an `ISTIO_VERSION` environment variable that defines the {istio} version to install on all the clusters by running the following command:
++
+[source,terminal]
+----
+$ export ISTIO_VERSION=1.24.1
+----
+
+. Create a `REMOTE_CLUSTER_NAME` environment variable that defines the name of the cluster by running the following command:
++
+[source,terminal]
+----
+$ export REMOTE_CLUSTER_NAME=cluster1
+----
+
+. Set up the environment variable that contains the `oc` command context for the control plane cluster by running the following command:
++
+[source,terminal]
+----
+$ export CTX_CONTROL_PLANE_CLUSTER=<context_name_of_the_control_plane_cluster>
+----
+
+. Set up the environment variable that contains the `oc` command context for the data plane cluster by running the following command:
++
+[source,terminal]
+----
+$ export CTX_DATA_PLANE_CLUSTER=<context_name_of_the_data_plane_cluster>
+----
+
+. Set up the ingress gateway for the control plane:
+
+.. Create a project called `istio-system` by running the following command:
++
+[source,terminal]
+----
+$ oc get project istio-system --context "${CTX_CONTROL_PLANE_CLUSTER}" || oc new-project istio-system --context "${CTX_CONTROL_PLANE_CLUSTER}"
+----
+
+.. Create an `Istio` resource on the control plane cluster to manage the ingress gateway by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc --context "${CTX_CONTROL_PLANE_CLUSTER}" apply -f -
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v${ISTIO_VERSION}
+  namespace: istio-system
+  global:
+    network: network1
+EOF
+----
+
```

Comment:
Create an IstioCNI resource on the data plane cluster by running the following command:


$ cat <<EOF | oc --context "${CTX_CONTROL_PLANE_CLUSTER}" apply -f -
apiVersion: sailoperator.io/v1
kind: IstioCNI
metadata:
  name: default
spec:
  version: v${ISTIO_VERSION}
  namespace: istio-cni
EOF